backend/python/transformers/backend.py

```diff
@@ -77,18 +74,19 @@ def LoadModel(self, request, context):
         """
         model_name = request.Model
 
-        compute = "auto"
+        compute = torch.float16
         if request.F16Memory == True:
             compute=torch.bfloat16
 
-        self.CUDA = request.CUDA
+        self.CUDA = torch.cuda.is_available()
```

Comment:
would make sense probably to keep reading `request.CUDA` ? maybe someone might want to turn it of explicitly afterall. Not a blocker

Reply:
It definitely make sense but request.CUDA by default is false, this means that it should be always set to true to use it. I think this way is more user friendly.
I can change if you want: np.